xbmc/interfaces/Builtins.cpp

```diff
@@ -1843,14 +1847,28 @@ int CBuiltins::Execute(const std::string& execString)
     CSettings::Get().SetBool("debug.showloginfo", !debug);
     g_advancedSettings.SetDebugMode(!debug);
   }
+  //TODO deprecated. To be replaced by pvr.startmanager
   else if (execute == "startpvrmanager")
   {
     g_application.StartPVRManager();
   }
+  else if (execute == "pvr.startmanager")
```

Comment:
Does it really make any difference? If this is merged (don't know if still for Isengard but likely J*?) the first thing I'll do is to remove them anyway